public/index.html

```diff
@@ -277,11 +320,17 @@
Each((function(e){n+=e.username+\"\\n\"})),[4,navigator.clipboard.writeText(n)];case 1:return r.sent(),alert(\"List copied to clipboard!\"),[2]}}))}))},t.getMaxPage=function(e){var t=Math.ceil(e.length/l.UNFOLLOWERS_PER_PAGE);return t<1?1:t},t.getCurrentPageUnfollowers=function(e,t){return a([],e,!0).sort((function(e,t){return e.username>t.username?1:-1})).splice(l.UNFOLLOWERS_PER_PAGE*(t-1),l.UNFOLLOWERS_PER_PAGE)},t.getUsersForDisplay=function(e,t,n,r,o){for(var a=[],u=function(e){var u=void 0!==t.find((function(t){return t.id===e.id}));switch(n){case\"non_whitelisted\":if(u)return\"continue\";break;case\"whitelisted\":if(!u)return\"continue\";break;default:i()}if(!o.showPrivate&&e.is_private)return\"continue\";if(!o.showVerified&&e.is_verified)return\"continue\";if(!o.showFollowers&&e.follows_viewer)return\"continue\";if(!o.showNonFollowers&&!e.follows_viewer)return\"continue\";if(!o.showWithOutProfilePicture&&e.profile_pic_url.includes(l.WITHOUT_PROFILE_PICTURE_URL_ID))return\"continue\";var c=e.username.toLowerCase().includes(r.toLowerCase())||e.full_name.toLowerCase().includes(r.toLowerCase());if(\"\"!==r&&!c)return\"continue\";a.push(e)},c=0,s=e;c<s.length;c++)u(s[c]);return a},t.getUnfollowLogForDisplay=function(e,t,n){for(var r=[],o=0,a=e;o<a.length;o++){var l=a[o];if((n.showSucceeded||!l.unfollowedSuccessfully)&&(n.showFailed||l.unfollowedSuccessfully)){var i=l.user.username.toLowerCase().includes(t.toLowerCase());(\"\"===t||i)&&r.push(l)}}return r},t.assertUnreachable=i,t.sleep=function(e){return new Promise((function(t){setTimeout(t,e)}))},t.getCookie=u,t.urlGenerator=function(e){var t=u(\"ds_user_id\");return void 0===e?'https://www.instagram.com/graphql/query/?query_hash=3dec7e2c57367ef3da3d987d89f9dbc8&variables={\"id\":\"'.concat(t,'\",\"include_reel\":\"true\",\"fetch_mutual\":\"false\",\"first\":\"24\"}'):'https://www.instagram.com/graphql/query/?query_hash=3dec7e2c57367ef3da3d987d89f9dbc8&variables={\"id\":\"'.concat(t,'\",\"include_reel\":\"true\",\"fetch_mutual\":\"false\",\"first\":\"24\",\"after\":\"').concat(e,'\"}')},t.unfollowUserUrlGenerator=function(e){return\"https://www.instagram.com/web/friendships/\".concat(e,\"/unfollow/\")};var l=n(563);function i(e){throw new Error(\"Statement should be unreachable\")}function u(e){var t=\"; \".concat(document.cookie).split(\"; \".concat(e,\"=\"));return 2!==t.length?null:t.pop().split(\";\").shift()}}},t={};function n(r){var o=t[r];if(void 0!==o)return o.exports;var a=t[r]={id:r,exports:{}};return e[r].call(a.exports,a,a.exports,n),a.exports}n.n=e=>{var t=e&&e.__esModule?()=>e.default:()=>e;return n.d(t,{a:t}),t},n.d=(e,t)=>{for(var r in t)n.o(t,r)&&!n.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},n.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),n.r=e=>{\"undefined\"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:\"Module\"}),Object.defineProperty(e,\"__esModule\",{value:!0})},n.nc=void 0,n(221)})();";//__END_OF_SCRIPT__
```

Comment:
Hi, this part is auto generated by a github action so please dont change it on the PR or at least the should end in //__END_OF_SCRIPT__ like was before.